data_dictionary/data_dictionary.cc

```diff
@@ -415,6 +415,8 @@ cql3::description keyspace_metadata::describe(const replica::database& db, cql3:
                 os << " AND tablets = {'enabled': false}";
             } else if (_initial_tablets.value() > 0) {
                 os << " AND tablets = {'initial': " << _initial_tablets.value() << "}";
+            } else {
+                os << " AND tablets = {'enabled': true}";
```

Comment:
printing out if "tablets" is "enabled" if it is enabled breaks the existing tests. see https://jenkins.scylladb.com/job/scylla-master/job/next/8340/testReport/junit/cqlsh_tests.cqlsh_tests/TestCqlsh/Build___x86___dtest_with_topology_changes___test_describe/ . 

because https://github.com/scylladb/scylla-dtest/blob/bbd917ceebf01ad7a0185b5ce9442c0ed2e50a57/cqlsh_tests/cqlsh_tests.py#L754 does not include this option yet in its `CREATE KEYSPACE` statement.

if we want to make this explicit, probably we can update `TestCqlsh.execute()` so that it accepts multiple `expected_output`, if any of them matches, we can consider the test pass? and after this change in scylla lands, we can change `TestCqlsh.execute()`, so that it only accepts the `expected_output`  containing `AND tablets ...`.

Reply:
https://github.com/scylladb/scylla-dtest/pull/5128